.eslintrc.js

```diff
@@ -4,12 +4,16 @@ module.exports = {
     node: true
   },
   'extends': [
-    'plugin:vue/essential',
+    'eslint:recommended',
+    'plugin:vue/recommended',
```

Comment:
Is it possible to change extends to  '@vue/airbnb'? Its more common-used style guide